packages/webcert/src/feature/subscription/SubscriptionWarningModal.test.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import { Provider } from 'react-redux'
+import { configureStore, EnhancedStore } from '@reduxjs/toolkit'
+import reducer from '../../store/reducers'
+import { apiMiddleware } from '../../store/api/apiMiddleware'
+import { userMiddleware } from '../../store/user/userMiddleware'
+import dispatchHelperMiddleware, { clearDispatchedActions } from '../../store/test/dispatchHelperMiddleware'
+import { createMemoryHistory } from 'history'
+import SubscriptionWarningModal from './SubscriptionWarningModal'
+import { updateUserResourceLinks } from '../../store/user/userActions'
+import { getSubscriptionWarningResourceLink } from '@frontend/common'
+import userEvent from '@testing-library/user-event'
+
+let testStore: EnhancedStore
+const history = createMemoryHistory()
+history.push = jest.fn()
```

Comment:
Why are we mocking history.push() for these tests? Is it just copy/paste from another file?

Reply:
Yes, copy/paste :) I fix.